openshift/openshift-ansible-contrib
# Moving DNS roles (generate views and records) out of playbook and into roles area

#### What does this PR do?
As the `openstack_dns_views` and `openstack_dns_records` are now generic (i.e.: not openstack specific), and are really a collection of tasks, they are being moved out of the playbook area and into 2 roles for easier re-use. 

#### How should this be manually tested?
Run a provisioning with internal or external DNS server; make sure there are no errors and all DNS records exists as expected. 

#### Is there a relevant Issue open for this?
N/A

#### Who would you like to review this?
cc: @tomassedovic @bogdando
